docs/l10n.rst

```diff
@@ -452,13 +452,70 @@ When a request is made for a particular page, bedrock's rendering function
 automatically checks for a locale-specific template, and, if one exists, will
 render it instead of the originally specified (locale-agnostic) template.
 
+.. NOTE::
+
+    Creating a locale-specific template for en-US was not possible when this
+    feature was introduced, but it is now. So you can create your en-US-only
+    template and the rest of the locales will continue to use the default.
+
 .. IMPORTANT::
 
     Note that the presence of an L10n template (e.g.
     ``locale/de/templates/firefox/new.html``) will take precedence over
     a locale-specific template in bedrock.
 
 
+Specifying Active Locales in Views
+----------------------------------
+
+Normally we rely on activation tags in our translation files (.lang files)
+to determine in which languages a page will be available. This will almost always
+be what we want for a page. But sometimes we need to explicitly state the locales
+available for a page. The `impressum` page for example is only available in German
+and the template itself has German hard-coded into it since we don't need it to be
+translated into any other languages. In cases like these we can send a list of locale
+codes with the template context and it will be the final list. This can be accomplished
+in a few ways depending on how the view is coded.
+
+For a plain view function, you can simply pass a list of locale codes to `l10n_utils.render`
+in the context using the name `active_locales`. This will be the full list of available
+translations. Use `add_active_locales` if you want to add languages to the existing list:
+
+.. code-block:: python
+
+    def french_and_german_only(request):
+        return l10n_utils.render(request, 'home.html', {'active_locales': ['de', 'fr'])
+
+If you don't need a custom view and are just using the `page()` helper function in your `urls.py`
+file, then you can similarly pass in a list:
+
+.. code-block:: python
+
+    page('about', 'about.html', active_locales=['en-US', 'es-ES']),
```

Comment:
In this case would we also have to put the strings in for translation to Spanish in the traditional way? And would we not be able to deploy the page until all the strings are translated in Ponton?

Reply:
Or this some kind of magical theoretical page that is appropriate to display in both without translation?

Reply:
Further reading suggests different content will be served for each locale but I'm not able to figure out where one should store that different content to have it delivered. More documentation required maybe?

Reply:
The idea is that it would be used with locale-specific templates that you just hard-code in the language of the template. So it's basically saying that the dev team is taking responsibility for the translation of these pages. It should rarely be used, but it's a means to prevent the l10n team from having to create empty .lang files with the `## active ##` tag at the top.